app/src/main/java/com/example/mvp9/data/UserRepostitory.java

```diff
@@ -0,0 +1,34 @@
+package com.example.mvp9.data;
+
+import com.example.mvp9.data.local.UserDataSource;
+import com.example.mvp9.model.User;
+
+public class UserRepostitory implements UserDataSource {
+    private UserDataSource mUserDataLocal;
+    private static  UserRepostitory instance;
+    public static UserRepostitory getInstance(UserDataSource userDataSource)
+    {
+        if(instance==null){
```

Comment:
```suggestion
        if(instance==null) {
```
Format lại code cho đúng convention, sử dụng ctrl + alt + L